opm/input/eclipse/EclipseState/Grid/EclipseGrid.cpp

```diff
@@ -2096,29 +2310,55 @@ std::vector<double> EclipseGrid::createDVector(const std::array<int,3>& dims, st
 
         for (std::size_t index = 0; index < lgr_input.size(); index++) {
             const auto lgr_cell = lgr_input.getLgr(index);
+<<<<<<< HEAD
```

Comment:
Could this be cleaned up?

Reply:
I have already cleared of all these. For some unknown reason git has corrupted the rebase and some of these were "spread". Moreover, git did output a message confirm the rebase that been successful. I have taken care of everything and tested the code.

Reply:
I'm sorry but somebody has to speak for the tool :P git doesn't do anything you don't tell it to. 

in particular, when you rebase you rewrite commits to remove conflicts. iow, there should be no additional commits after a rebase. if there are, the wielder did it wrong, you cannot blame the tool.

Reply:
@akva2 of course that was a misuse of the tool on my side. I think that was explicit.

Reply:
hehe yes, my point was meant to be half witty and half informative.

my point is that when you have commits like 'rebased code'. this means you already have screwed up the rebase. you need to rebase again and remove those cause those commits is not something we want in the git history. you added another set of commits fixing up the already existing screwups. that is not the way to go about it.

Reply:
@akva2 , thank you so much for your help and I promise I will take in consideration from now on.. Indeed, I struggled a bit with the rebasing but this was due to merging two different PRs of mine exclusively.  have just met with @bska and he helped me verify the code to ensure everything is okay.